pkg/scheduler/partition.go

```diff
@@ -582,7 +582,7 @@ func (pc *PartitionContext) updatePartitionResource(delta *resources.Resource) {
 		} else {
 			pc.totalPartitionResource.AddTo(delta)
 		}
-		pc.root.SetMaxResource(pc.totalPartitionResource)
+		pc.root.SetMaxResource(pc.totalPartitionResource.Clone())
```

Comment:
not sure whether this `clone` is required since `SetMaxResource` do clone also (https://github.com/apache/yunikorn-core/blob/78862645c7e3f790b4bf2543865ba5a85c115163/pkg/scheduler/objects/queue.go#L1279)

Reply:
Ah cool. Good catch. I'll undo this.